state_solver/rear_tire.py

```diff
@@ -60,4 +64,11 @@ def steered_inclination_angle_gain(self, steered_angle):
 
     @property
     def tube_geometry(self):
-        return self.params.rear_tube_normals, self.params.rear_lever_arms
\ No newline at end of file
+        return self.params.rear_tube_normals, self.params.rear_lever_arms
+
+    @property
+    def radius(self):
+        return self.params.rear_tire_radius
+
+    def get_slip_ratio(self, slip_ratios):
```

Comment:
confused here, why are slip_ratios being passed as attribte to get_slip_ratio?

Reply:
Its pretty dirty... slip ratios are passed in as 0, 1, 2, 3 because they're sweeped dependent states and not output states. If you can find a better way of doing it lmk